Modules/Devices/test/Devices.Application.Tests/Tests/Tiers/Commands/DeleteTier/HandlerTests.cs

```diff
@@ -5,41 +5,31 @@
 using Backbone.Modules.Devices.Domain.Aggregates.Tier;
 using Backbone.Modules.Devices.Domain.Entities;
 using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
-using Enmeshed.DevelopmentKit.Identity.ValueObjects;
 using FakeItEasy;
 using FluentAssertions;
-using Microsoft.Extensions.Logging;
 using Xunit;
 using ApplicationException = Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
 
 namespace Backbone.Modules.Devices.Application.Tests.Tests.Tiers.Commands.DeleteTier;
 public class HandlerTests
 {
-    private readonly ITiersRepository _tierRepository;
+    private readonly ITiersRepository _tiersRepository;
     private readonly IEventBus _eventBus;
     private readonly Handler _handler;
 
     public HandlerTests()
     {
-        _tierRepository = A.Fake<ITiersRepository>();
+        _tiersRepository = A.Fake<ITiersRepository>();
         _eventBus = A.Fake<IEventBus>();
         _handler = CreateHandler();
     }
 
-    private Handler CreateHandler()
-    {
-        var logger = A.Fake<ILogger<Handler>>();
-        return new Handler(_tierRepository, logger, _eventBus);
-    }
-
     [Fact]
     public async Task Cannot_Delete_Basic_Tier()
     {
         // Arrange
-        var basicTierName = TierName.Create("basic-tier").Value;
-        var basicTier = new Tier(basicTierName);
-        A.CallTo(() => _tierRepository.GetBasicTierAsync(A<CancellationToken>._)).Returns(Task.FromResult(basicTier));
-        A.CallTo(() => _tierRepository.FindById(basicTier.Id, A<CancellationToken>._)).Returns(Task.FromResult(basicTier));
+        var basicTier = new Tier(TierName.Create(TierName.BASIC_DEFAULT_NAME).Value);
```

Comment:
Remember to update this after you changed the type of the BASIC_DEFAULT_NAME. The compiler won't complain because it's still valid code due to the implemented implicit conversion from TierName to string.